geoportal/geoportailv3_geoportal/static-ngeo/js/draw/featurepopup.html

```diff
@@ -24,10 +24,10 @@
       </div>
     </div>
     <span ng-show="ctrl.featureElevation" translate>Elevation: {{ ctrl.featureElevation }}</span>
-    <div class="feature-popup-profile" ng-show="ctrl.showFeatureProfile.active">
+    <div class="feature-popup-profile" ng-if="ctrl.showFeatureProfile.active"> <!-- us-ng-if otherwise profile is duplicated in dom... -->
```

Comment:
```suggestion
    <div class="feature-popup-profile" ng-if="ctrl.showFeatureProfile.active"> <!-- use ng-if otherwise profile is duplicated in dom... -->
```